<?xml version="1.0"?>
<sdf version="1.7">
  <model name="simple_mug">
    <link name="main_body">
      <inertial>
        <pose>0.01 0 0.05 0 0 0</pose>
        <mass>0.094</mass>
        <inertia>
          <ixx>0.000156</ixx>
          <ixy>0</ixy>
          <ixz>0</ixz>
          <iyy>0.000156</iyy>
          <iyz>0</iyz>
          <izz>0.00015</izz>
        </inertia>
      </inertial>
      <collision name="main_body_collision">
        <pose>0 0 0.05 0 0 0</pose>
        <geometry>
          <cylinder>
            <length>0.1</length>
            <radius>0.04</radius>
          </cylinder>
        </geometry>
        <surface>
          <friction>
            <ode>
              <mu>0.9</mu>
              <mu2>0.5</mu2>
            </ode>
          </friction>
        </surface>
      </collision>
      <visual name="main_body_visual">
        <pose>0 0 0.05 0 0 0</pose>
        <geometry>
          <cylinder>
            <length>0.1</length>
            <radius>0.04</radius>
          </cylinder>
        </geometry>
      </visual>
      <visual name="mug_handle_visual">
        <pose>0.055 0 0.05 0 0 0</pose>
        <geometry>
          <box>
            <size>0.03 0.02 0.08</size>
          </box>
        </geometry>
      </visual>
    </link>
  </model>
</sdf>
